README.md

```diff
@@ -40,7 +43,7 @@ LOGZIO_ACCOUNT_TOKEN=<ACCOUNT-TOKEN> \
 LOGZIO_LISTENER_HOST=<LISTENER-URL> \
 LOGZIO_API_TOKEN=<API-TOKEN> \
 LOGZIO_API_HOST=<API-URL> \
-java -Dloader.path='zipkin-logzio.jar,zipkin-logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
+java -Dloader.path='logzio.jar,logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
```

Comment:
conventionally all of our storage modules use simplified form as when you add many it gets tiring to redundantly prefix